src/Support/helpers.php

```diff
@@ -926,6 +964,7 @@ function title_case($value)
      */
     function resolve_path($path)
     {
+        $path = str_replace('/', DIRECTORY_SEPARATOR, $path);
```

Comment:
I replaces slashes here as well, is it ok?

Reply:
@mjauvin The PathResolver already normalises the slashes, so it doesn't need this line. I'll change it up with my fixes.